### PR TITLE
Better name formatting for TeamCity

### PR DESCRIPTION
**Pull request by ronniebarker**
#153 - maybe like this...

### CODE REVIEW

File: Expecto/Expecto.fs [hunk 1]
```diff
@@ -736,14 +739,14 @@ module Impl =
...
-            "name", n ] }
+            "flowId", ( formatName n )
+            "name", ( formatName n ) ] }
 
         passed = fun n d -> async {
           do! innerPrinter.passed n d
           tcLog "testFinished" [
-            "flowId", n
-            "name", n
+            "flowId", ( formatName n )
```
Comment (haf) [inline Expecto/Expecto.fs]: Remove the parenthesis and the extra spaces.

**Comment by haf** [comment]
This feature will interacts with https://github.com/haf/expecto/issues/146 – with #146 we'd not need string-replace of the already-*joined* name, only for each string in the `name` field of the Message.

**Comment by ronniebarker** [comment]
Are you suggesting that #146 makes this irrelevant or that they can work together?

**Comment by haf** [comment]
They can work together.

**Comment by haf** [comment]
Thanks @ronniebarker !

**Comment by haf** [comment]
https://github.com/haf/expecto/releases/tag/v5.0.0-rc1

**Pull request merged**